src/profile/v100/networks/network/network_id.rs

```diff
@@ -114,40 +114,40 @@ impl std::fmt::Display for NetworkID {
 }
 
 impl NetworkID {
-    /// Looks up a `NativeNetworkDefinition` in lookup table,
+    /// Looks up a `ScryptoNetworkDefinition` in lookup table,
     /// this is used internally for radix_engine_common::address::AddressBech32Decoder,
     /// and to read out the canonical name (logical name) for a network.
-    pub(crate) fn network_definition(&self) -> NativeNetworkDefinition {
+    pub(crate) fn network_definition(&self) -> ScryptoNetworkDefinition {
         use NetworkID::*;
         match self {
-            Mainnet => NativeNetworkDefinition::mainnet(),
-            Stokenet => NativeNetworkDefinition::stokenet(),
-            Adapanet => NativeNetworkDefinition::adapanet(),
-            Nebunet => NativeNetworkDefinition::nebunet(),
-            Kisharnet => NativeNetworkDefinition::kisharnet(),
-            Ansharnet => NativeNetworkDefinition::ansharnet(),
-            Zabanet => NativeNetworkDefinition::zabanet(),
-            Enkinet => NativeNetworkDefinition {
+            Mainnet => ScryptoNetworkDefinition::mainnet(),
+            Stokenet => ScryptoNetworkDefinition::stokenet(),
+            Adapanet => ScryptoNetworkDefinition::adapanet(),
+            Nebunet => ScryptoNetworkDefinition::nebunet(),
+            Kisharnet => ScryptoNetworkDefinition::kisharnet(),
+            Ansharnet => ScryptoNetworkDefinition::ansharnet(),
+            Zabanet => ScryptoNetworkDefinition::zabanet(),
+            Enkinet => ScryptoNetworkDefinition {
```

Comment:
Don't such definitions exist in RET? This is why you added some custom ones?

Reply:
mm does not exist